netbox/core/tables/tasks.py

```diff
@@ -0,0 +1,86 @@
+import django_tables2 as tables
+from django_tables2.utils import A  # alias for Accessor
+from django.urls import reverse
+from django.utils.html import mark_safe
+from django.utils.translation import gettext_lazy as _
+
+from netbox.tables import BaseTable, NetBoxTable, columns
+from utilities.templatetags.helpers import annotated_date
+from ..models import Job
```

Comment:
Remove unused imports.
```suggestion
from netbox.tables import BaseTable
from utilities.templatetags.helpers import annotated_date
```